roles/backup/README.md

```diff
@@ -0,0 +1,84 @@
+Role Name
+=========
+
+The purpose of this role is to create a backup of your AWX deployment.  This includes:
+  - backup of the postgresql database
+  - secret_key
+  - custom user config files
+  - manual projects
+
+
+Requirements
+------------
+
+This role assumes you are authenticated with an Openshift or Kubernetes cluster which:
+  - The awx-operator has been deployed to
+  - AWX is deployed to via the operator
+
+
+Usage
+----------------
+
+Then create a file named `backup-awx.yml` with the following contents:
+
+```yaml
+---
+apiVersion: awx.ansible.com/v1beta1
+kind: Backup
+metadata:
+  name: awx
+  namespace: my-namespace
+```
+
+Finally, use `kubectl` to create the awx instance in your cluster:
```

Comment:
To create the backup CR ? the awx instance is already there :)

Reply:
The idea here is that the AWX CR object has already been made and AWX is already deployed when a backup is performed.  So to perform a backup, the user will create a new backup CRO (using the same awx.ansible.com/v1beta1 API), which will then kick off the backup role.